tests/components/timer/test_init.py

```diff
@@ -237,7 +237,7 @@ def fake_event_listener(event):
             "call": SERVICE_CHANGE,
             "state": STATUS_ACTIVE,
             "event": EVENT_TIMER_CHANGED,
-            "data": {CONF_DURATION: 15},
+            "data": {CONF_DURATION: -5},
```

Comment:
May be it is better to place these exceptions in a separate test. The current tests should prove everything is still working as expected. In a followup we could refactor the tests.